conda/cli/main_info.py

```diff
@@ -186,6 +193,12 @@ def execute(args, parser):
         requests_version = "Error %s" % e
 
     try:
+        cenv = [p for p in itervalues(root_pkgs) if p['name'] == 'conda-env']
```

Comment:
Now I'm not letting it slide....

NoneType has no attribute "__iter__"

Do I have that memorized correctly yet?